File: fib.erl
-module(fib).

-export([start/1]).

fib(N)->
	case N of
		0->0;
		1->1;
		_->
			fib(N-1)+fib(N-2)
	end.

start(N)->
	Result=fib(N),
	io:format("Result: ~p~n",[Result]).

File: fib.js
var util = require("util")

function fib(num){
	if(num==0) return 0;
	if(num==1) return 1;
	return fib(num-1) + fib(num-2)
}

var count = process.argv[2] || 0;
var result = fib(parseInt(count));

var str = util.format("result: %d",result)
console.log(str)
